### PR TITLE
[MOD-2372] Recompile ti.paint for SDK 7.0.0 compatability

### PR DESCRIPTION
**Pull request by infosia**
JIRA ticket: [MOD-2372](https://jira.appcelerator.org/browse/MOD-2372)

Upcoming Titanium SDK 7.0.0 we are expecting module apiversion update. We need to bump up the version. SDK 7.0.0 will also include JavaScriptCore update, so we need to update cmake project too.

### CODE REVIEW

File: windows/manifest [hunk 1]
```diff
@@ -1,5 +1,5 @@
-version: 2.0.0
-apiversion: 3
+version: 2.1.0
```
Comment (garymathews) [inline windows/manifest]: Should be major, `3.0.0`

File: windows/cmake/FindJavascriptCore.cmake [hunk 2]
```diff
@@ -1,61 +1,40 @@
-# HAL
+# FindJavaScriptCore
+# Author: Chris Williams
 #
-# Copyright (c) 2017 Axway All Rights Reserved.
+# Copyright (c) 2014 by Appcelerator, Inc. All Rights Reserved.
```
Comment (garymathews) [inline windows/cmake/FindJavascriptCore.cmake]: Should be `2017 Axway, Inc.` ?

File: windows/manifest (hunk 1 above)
Comment (infosia) [reply windows/manifest]: Updated 👍

**Comment by ao1ansal** [comment]
The latest Android version, is not working when we add the Paint view inside  a scrollview. I am using ti.paint-android-4.0.0.zip from [](https://github.com/appcelerator-archive/ti.paint/releases) . Titanium SDK 7.0.1 GA and Appcelerator Studio 5.0.0. ti.paint will work fine if this is not inside of a scrollview. Please provide a fix asap or a work around.

**Pull request merged**